Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM python:3.6-alpine
 
-ENV PYTHONPATH=./.pip:/app/.pip:.: \
-    DOCKER=True
-COPY . /app/
-RUN python3 -m pip install -r /app/requirements/requirements.txt -t /app/.pip
+RUN apk add --update --no-cache gcc g++ && pip install dumb-init
 
+COPY sirbot/ /app/sirbot/
```

Comment:
Instead of needing to prefix the `/app/` for each of these, consider using `WORKDIR /app`, it will auto-create the directory and then all copies and commands are relative to that directory.

This isn't taking any advantage of caching - you may want to try out something like:

```
FROM python:3.6-alpine
RUN apk add --update --no-cache gcc g++ && pip install dumb-init
WORKDIR /app
COPY requirements.txt .
RUN pip install -r requirements.txt  # note this will cache the layer and only re-build it if the `requirements.txt` changes or an older layer is invalidated
COPY . .  # copy the rest

ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
CMD ["python3", "run.py"]
```

You'll notice I added the `dumb-init` entry point too :)
```